site/content/docs/5.3/about/brand.md

```diff
@@ -20,15 +20,15 @@ Have a need for Boosted's brand resources? Great! We have only a few guidelines
   <div class="col">
     <div class="ratio ratio-1x1">
     <figure class="d-flex bg-black">
-      <img src="/docs/{{< param docs_version >}}/assets/brand/orange-logo.svg" class="figure-img img-fluid m-auto" width="220" height="220" role="img" alt="Orange" loading="lazy">
+      <img src="/docs/{{< param docs_version >}}/assets/brand/orange-logo.svg" class="figure-img img-fluid m-auto" width="220" height="220" role="img" alt="Orange full logo" loading="lazy">
```

Comment:
```suggestion
      <img src="/docs/{{< param docs_version >}}/assets/brand/orange-logo.svg" class="figure-img img-fluid m-auto" width="220" height="220" role="img" alt="Orange master logo" loading="lazy">
```
I think I'd prefer to ear this rather than full.